docs/app/api_spec.md

```diff
@@ -102,10 +105,81 @@ Note: we will defer making any of these required for now
 }
 ```
 
+## Endpoint: `POST /feedback`
+
+### Description
+
+This endpoint accepts feedback from a navigator and responds with the user_id, value, step_id and comment given for the feedback request.
+
+### Request Parameters
+
+**Headers**
+
+- **Authorization**: Bearer token for authenticating API access.
+
+**Parameters**
+
+- **user_id** _(string, optional)_: Unique identifier for the navigator within ImagineLA.
+- **comment** _(string, optional)_: The user's feedback comment for the LLM response.
+
+- **session_id** _(string, required)_: Unique identifier for the current session, to track prior messages in the conversation.
+- **response_id** _(string, required)_: Unique identifier for the chatbot response.
+- **is_positive** _(bool, required)_: If the chatbot response is helpful to the navigator or not.
+
+---
+
+### Responses
+
+#### Success Response (200 OK)
+
+**Body**
+
+```json
+{
+  "user_id": "string",       // Unique identifier for the navigator within ImagineLA
+  "value": "float",          // 1 if is_positive was "true" and 0 if is_positive was "false"
+  "step_id": "string",       // ID of the step associated with the score
+  "comment": "string"        // Navigator's comment on the response
```

Comment:
Why are these sent back to the client? They're not used for anything, right?

Reply:
FWIW I also wondered about this -- `step_id` seems to always match `response_id`. Could also just return an empty 200 OK response to verify the feedback was received.